packages/app-project/src/helpers/theme/README.md

```diff
@@ -1,5 +1,3 @@
 # Theme
 
 Creates a Grommet theme by performing a deep merge of the Grommet base theme with the Zooniverse custom Grommet theme.
-
-This was originally done in the `GrommetWrapper` theme, but was moved here so it can be reused with `@artsy/fresnel` to extract the breakpoints for responsive sizing.
```

Comment:
`@artsy/fresnel` was removed awhile ago. This helper could probably be removed, but will do so in different PR.